CONTRIBUTING.md

```diff
@@ -51,7 +47,6 @@ Just before submitting a PR, run `npm run create-readme` to generate the new REA
   * Use [./.README/rules/require-valid-file-annotation.md](./.README/rules/require-valid-file-annotation.md) as a template.
   * Ensure that rule documentation document includes `<!-- assertions spaceAfterTypeColon -->` declaration.
 1. Update [./.README/README.md](/.README/README.md) to include the new rule.
-
-A CI service will build and publish the new documentation.
```

Comment:
I am not sure about this change, however, there doesn't seem to be this kind of automation. 🤔